packages/@sanity/portable-text-editor/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "@sanity/portable-text-editor",
+  "version": "2.14.0",
+  "description": "Portable Text Editor made in React",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
```

Comment:
this should be changed to `./dist/dts/index.d.ts`

Reply:
Fixed!